WordPress/WordPressShareExtension/AppExtensionsService.swift

```diff
@@ -102,6 +102,17 @@ extension AppExtensionsService {
         })
     }
 
+    func isAuthorizedToUploadMedia(in sites: [RemoteBlog], for selectedSiteID: Int) -> Bool {
+        let siteID = NSNumber(value: selectedSiteID)
+        var isAuthorizedToUploadFiles = false
+        for site in sites {
```

Comment:
I am not sure if this would make sense, but I think this loop could be avoided by doing something similar to this:

``
isAuthorizedToUploadFIles = sites.filter({$0.blogId == siteID}).isUserCapableOf(.uploadFiles)
``

That way, we could not only avoid the loop, but declare isAuthorizedToUploadFiles as a `let` instead of a `var`